README.md

```diff
@@ -143,7 +143,7 @@ may be able to remove the `linker = "clang"` line.
 
 ```toml
 [target.x86_64-unknown-linux-gnu]
-rustflags = ["-C", "link-arg=-fuse-ld=/path/to/mold"]
+rustflags = ["-C", "link-arg=-fuse-ld=mold"]
```

Comment:
It seems that gcc doesn't support the path, hence I would change `may be able to remove the `linker = "clang"` line.` to `may be able to remove the `linker = "clang"` line and change the '-fuse-ld=' setting to 'mold'.`